features/datachoices/pom.xml

```diff
@@ -38,6 +38,16 @@
             <artifactId>oshi-core</artifactId>
             <version>${oshi.core.version}</version>
         </dependency>
+            <dependency>
```

Comment:
we have a maven module that already maintains `jna-dependencies` use that here.
```
  <groupId>org.opennms.dependencies</groupId>
  <artifactId>jna-dependencies</artifactId>
```
nit: formatting